src/test/com/fulcrologic/fulcro/application_spec.cljc

```diff
@@ -25,22 +32,103 @@
-      (= result (eql/ast->query (app/default-global-eql-transform (eql/query->ast query))))
-      [:ui/name :x] [:x]
+      (= result (remove #{:com.wsscode.pathom.core/errors} 
+                        (eql/ast->query (app/default-global-eql-transform (eql/query->ast query)))))
+      [:ui/name :x]  [:x]
       '[(:ui/name {:b 2}) :x] [:x]
       [{:ui/name [:b]} :x] [:x]
       [[:ui/name 42] :x] [:x]
       [{[:ui/name 42] [:t]} :x] [:x]
       [:ui/name :x :ui/adsf-b] [:x]))
   (behavior "Removes ui and fulcro keys from mutation join queries, but NOT mutation params"
-    (are [query result]
+    (are [query result] ; FIXME fails due to added :tempids
       (= result (eql/ast->query (app/default-global-eql-transform (eql/query->ast query))))
       [{'(f {:ui/param 1 :com.fulcrologic.fulcro/param 42}) [:ui/a :b {:com.fulcrologic.fulcro.core/boo [:y]}]}]
       [{'(f {:ui/param 1 :com.fulcrologic.fulcro/param 42}) [:b]}]))
   (behavior "Removes items that are namespaced to Fulcro itself"
     (are [query result]
-      (= result (eql/ast->query (app/default-global-eql-transform (eql/query->ast query))))
+      (= result (remove #{:com.wsscode.pathom.core/errors} 
+                        (eql/ast->query (app/default-global-eql-transform (eql/query->ast query)))))
       [{[::uism/asm-id 42] [:y]} :x] [:x]
       [::uism/asm-id :x] [:x]
       [{::uism/asm-id [:y]} :x] [:x]
       [::dr/id ::dr/current-route [::uism/asm-id '_] :x] [:x])))
+
+(defn sort-nested 
+  "Sort keywords in vectors anywhere in the query to simplify comparison"
+  [query]
+  (letfn [(compare-any [x y]
+            (if (every? keyword? [x y])
+              (compare x y)
+              (if (keyword? x) -1 1)))]
+    (walk/postwalk
+      #(cond-> %
+         (vector? %) (->> (sort compare-any) vec))
+      query)))
+
+(defn transform-query [query]
+  (-> query
+      eql/query->ast
+      rapp/default-global-eql-transform
+      eql/ast->query))
+(-> (eql/query->ast '[{(my-mutation) [:x]}]) :children first)
+(defn superset?*
+  "Takes an `expected` set, and will check that `actual` contains at least all the elements in it.
+   Eg:
+   ```
+   ; PASSES
+   #{:a :b} =check=> (superset?* #{:a})
+
+   ; FAILS
+   #{:a} =check=> (superset?* #{:a :b})
+   ```"
+  [expected]
+  ;(assert-is! `superset?* set? expected)
+  (_/and*
+    (_/is?* seqable?)
+    (_/checker [actual]
+      (when-not (set/subset? expected (set actual))
+        {:actual {:extra-values (set/difference expected (set actual))}
+         :expected `(~'superset?* ~expected)
+         :message "Found values missing from the set"}))))
+
+;; (letfn [(transform [query] (sort-nested (transform-query query)))]
+;;   (transform '[(mutation6)]))
+
+(specification "Default EQL transform additions"
+  (behavior "Adds ::p/errors to all queries BUT not mutations"
+    (letfn [(transform [query] (sort-nested (transform-query query)))]
+      (assertions
+        (transform [:a]) => [:a ::p/errors]
```

Comment:
These tests would be way clearer if you had a helper that took EQL, turned it into an AST, passed it to transform, then converted it *BACK* to EQL...then you could write something like

```
"Leaves regular mutations alone"
(transform '[(mutation)]) => '[(mutation)]
"Adds :tempids to mutation joins"
(transform '[{(mutation)] [:x]}) => '[{(mutation) [:x :tempids]}]
```